File: 040-049/ABC045_A.js
'use strict';

function Main(input) {
  const tmp = input
    .trim()
    .split('\n')
    .map((v) => parseInt(v, 10));

  const a = tmp[0];
  const b = tmp[1];
  const h = tmp[2];

  const result = ((a + b) * h) / 2;

  console.log(result);
}

Main(require('fs').readFileSync('/dev/stdin', 'utf8'));

File: 040-049/ABC045_A.c
// ABC045 A - 台形

#include <stdio.h>

int main(void)
{
    int a, b, h;
    int result = 0;

    // input
    scanf("%d", &a);
    scanf("%d", &b);
    scanf("%d", &h);

    result = (a + b) * h / 2;

    // output
    printf("%d\n", result);

    return 0;
}
